chia/types/coin_spend.py

```diff
@@ -25,3 +27,26 @@ def additions(self) -> List[Coin]:
 
     def reserved_fee(self) -> int:
         return fee_for_solution(self.puzzle_reveal, self.solution, INFINITE_COST)
+
+    def hints(self) -> List[bytes]:
```

Comment:
Can we write a comment saying this method should not be called in the full node?

Reply:
It is not safe

Reply:
ohh, we might want to move it to wallet utils?

Reply:
yeah that's a good idea